### PR TITLE
CLOUDP-95681: Split the command atlas events list

### PR DESCRIPTION
**Pull request by gssbzn**
_Jira ticket:_ CLOUDP-95681

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 *       @mongodb/mongocli
 
 # Docs maintained by Docs Cloud Team
-docs/*  @mongodb/docs-cloud-team
+/docs/  @mongodb/docs-cloud-team
```
Comment (gssbzn) [inline .github/CODEOWNERS:5]: @fmenezes 
```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
```
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners

File: e2e/cloud_manager/events_test.go [hunk 2]
```diff
@@ -32,12 +33,12 @@ func TestEvents(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	t.Run("ListProjectEvent", func(t *testing.T) {
+	t.Run("List Projec tEvent", func(t *testing.T) {
```
Comment (fmenezes) [inline e2e/cloud_manager/events_test.go:36]:
```suggestion
	t.Run("List Project Event", func(t *testing.T) {
```

**Review by fmenezes — APPROVED** [review]

File: e2e/cloud_manager/events_test.go (hunk 2 above)
Comment (gssbzn) [reply e2e/cloud_manager/events_test.go:36]: oops

**Review by fmenezes — APPROVED** [review]

File: internal/cli/events/list.go [hunk 3]
```diff
@@ -65,25 +81,27 @@ func (opts *ListOpts) Run() error {
...
-		Short:   "List events for an organization or project.",
+		Use:   "list",
+		Short: "Return all events for an organization or project.",
+		Deprecated: `  
+  To return project events prefer
+  $ mongocli atlas|ops-manager|cloud-manager events projects list [--projectId <projectId>]
+
+  To return organization events prefer
+  $ mongocli atlas|ops-manager|cloud-manager events organizations list [--orgId <orgId>]
+`,
```
Comment (melissamahoney-mongodb) [inline internal/cli/events/list.go:104]: Question: I don't think this currently displays in the RST output. Do we want it to?

**Review by melissamahoney-mongodb — APPROVED** [review]
One question, otherwise LGTM

File: internal/cli/events/list.go (hunk 3 above)
Comment (gssbzn) [reply internal/cli/events/list.go:104]: I think the current behaviour is to ignore deprecated commands, that works for me if it works for docs

Comment (melissamahoney-mongodb) [reply internal/cli/events/list.go:104]: Works for me.

**Pull request merged**